features/shop/checkout/shipping_order/seeing_shipping_method_when_at_least_one_unit_match_to_shipping_method.feature

```diff
@@ -64,8 +64,8 @@ Feature: Seeing shipping methods compatible with categories of units in my cart
     Scenario: Seeing shipping methods if some of my unit has variant with shipping category matching to the shipping category of shipping method
         Given the "T-Shirt banana" product's "M" size belongs to "Standard" shipping category
```

Comment:
```suggestion
        Given I have a product matching "Over-sized" shipping category in my cart
```
Just an idea for merging another two steps into one that focuses on what it is testing exactly.

Reply:
I like this idea as well, but to keep development moving, I’d suggest postponing these changes for now.